docs/docs/personalization.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Personalization
+---
+
+## What is Personalization?
+
+- Have you ever landed on a website that greeted you by name?
+
+- Did you ever search for a book and realized your suggested results based on your previous purchases?
+
+These are examples of personalization. Personalization adjusts the content of a website, and creates a custom experience for users. With personalization, one website can offer a variety of experiences to different visitors.
+
+## Why is Personalization Important?
+
+If you are [creating an e-commerce site](/tutorial/ecommerce-tutorial/), personalization is key in converting your visitors into buyers. Online merchants use personalization as a way to suggest relevant items to a shopper.
+
+Outside of e-commerce, personalization is important for other types of websites, like e-learning. Personalization, if implemented strategically, can help educational sites deliver more targeted learning experiences. For example, if a learner is struggling with the learning the basics of JavaScript, the website can give the learner additional JavaScript practice problems.
+
+## Using Personalization with Gatsby
+
+Personalization contains dynamic content. Personalization is possible in Gatsby because Gatsby can enhance static content by [React Hydration](/blog/2018-10-15-beyond-static-intro/#hydration). During build time Gatsby invokes the React server-side DOM to generate content.
```

Comment:
Hydration in relation to personalization was particularly challenging to describe. That being said, reading @gillkyle's [adding rehydration doc and updates to building apps overview PR](https://github.com/gatsbyjs/gatsby/pull/19214) was very helpful in helping me form a mental model of how hydration and rehydration works in Gatsby. 

If I'm missing any details about react hydration, or if my understanding of how Gatsby handles dynamic data is a little off, please let know!